__posts/2022-06-02-release-2.2.0.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: MDAnalysis 2.2 is out
+---
+
+We are happy to release the version 2.2.0 of MDAnalysis!
+
+In line with [NEP 29][], this version of MDAnalysis drops support for Python 3.7 and raises the minimum NumPy version to 1.19.0. Minimum version support has also been changed for the following packages; `networkx>=2.0`, `scipy>=1.5.0`, `gsd>=1.9.3`. Further details on MDAnalysis future support strategy and NEP29 will be released shortly.
+
+Supported Python versions: **3.8, 3.9, 3.10**
+
+Supported Operating Systems:
+  - **Linux**
+  - **Windows**
+  - **MacOS** (on [x86_64](https://en.wikipedia.org/wiki/X86-64) and for the first time on *[Apple M1](https://en.wikipedia.org/wiki/Apple_M1) ARM* architecture)
```

Comment:
Do we need to say that we don't have cf packages yet but that pip installs should work?